<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceAETemplateClass</class>
 <widget class="QMainWindow" name="ReplaceAETemplateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReplaceAETemplate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="replaceCharTab">
       <attribute name="title">
        <string>模板文字替换</string>
       </attribute>
       <attribute name="toolTip">
        <string>替换二进制模板文件中的指定字符串</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>查找</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>将要替换的字符串:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sourceCharLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="searchCharButton">
               <property name="text">
                <string>开始查找</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>查找结果:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="searchCharResult">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>说明：先输入要替换的字符串，程序将在二进制模板文件中进行查找。如果没有找到字符串，
则不能进行替换操作！</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>替换</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>替换的目标字符串:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="destCharLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="replaceCharButton">
               <property name="text">
                <string>开始替换</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>替换结果:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="replaceCharResult">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>条完成替换</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>说明：替换后即可打开模板进行查看。</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="replaceImageTab">
       <attribute name="title">
        <string>模板图片替换</string>
       </attribute>
       <attribute name="toolTip">
        <string>替换二进制模板文件中的指定图片</string>
       </attribute>
       <widget class="QTreeWidget" name="imageTreeWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>181</width>
          <height>311</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>9</y>
          <width>341</width>
          <height>311</height>
         </rect>
        </property>
        <property name="title">
         <string>操作面板</string>
        </property>
        <widget class="QLabel" name="imagePreviewLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>54</y>
           <width>371</width>
           <height>221</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>22</y>
           <width>321</width>
           <height>22</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>图片预览ID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="imageCountLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>321</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="chooseImageButton">
            <property name="text">
             <string>替换成...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="destImagePathLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="replaceImageButton">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionCloseFile"/>
    <addaction name="actionQuitApplication"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="actionDeveloper"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开模板文件...(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="text">
    <string>关闭文件(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuitApplication">
   <property name="text">
    <string>退出程序(&amp;Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDeveloper">
   <property name="text">
    <string>开发者(&amp;D)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="replaceaetemplate.qrc"/>
 </resources>
 <connections/>
</ui>
